### PR TITLE
Restructure main repository

### PR DESCRIPTION
**Pull request by jbelien**
Still in the same idea of simplifying the replication process and improving the documentation, I have restructured the main repository.

### City website/map

- All necessary files to build the city map (and website) have been moved to the `website/` directory
- Some files have been improved but no major changes
- Building script has completely been refactored (see [`build.js`](https://github.com/EqualStreetNames/equalstreetnames/blob/website/website/build.js))
  - Build script : `npm run build:belgium:brussels` (= `node build.js --city belgium/brussels`)
  - Serve script : `npm run build:belgium:brussels -- --serve` (= `node build.js --serve --city belgium/brussels`)
- ⚠️ **The restructure will have an impact on the deployment !**
  - I'll update all `deploy.yml` GitHub Actions workflows
  - For those who don't use GitHub Pages (@gislars @stalker314314 @ynux), you will have to run the build command from the `website/` directory and publish the `website/dist/belgium/brussels` directory (instead of `dist/belgium/brussels`) ; let me know if you want to switch to GitHub Pages deployment

### Data process

- All necessary files for the data process have been moved to the `process/` directory
- No changes, now you just need to run the `composer run update-data -- --city=belgium/brussels` in the `process/` directory (instead of the root directory)
- I'll update all `update-data.yml` GitHub Actions workflows (once this PR is merged)

### Global map

-  `index.html` file has been moved to the `global/` directory.
- That will allow me to upgrade that global map (in a near future)

### CODE REVIEW

**Comment by jbelien** [comment]
Any feedback is (more than) welcome @RobinLinde @gislars @stalker314314 @ynux !

**Comment by RobinLinde** [comment]
@jbelien I'm guessing there are some changes necessary in the city submodules, everything seems to work nicely after changing the index.html
```diff
-<link rel="stylesheet" href="../../../../../resources/style.scss">
+<link rel="stylesheet" href="../../../website/src/sass/style.scss">
```

```diff
-<script src="../../../../../resources/index.ts"></script>
+<script src="../../../website/src/script/index.ts"></script>
```
Will take a look at the other changes, but `package.json` looks a lot cleaner now 😃.

File: website/package.json [hunk 1]
```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@equalstreetnames/website",
+  "version": "1.0.0",
+  "description": "",
```
Comment (RobinLinde) [inline website/package.json:4]: Might be nice to add a description

File: website/package.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openknowledgebe/equalstreetnames.git"
+  },
+  "keywords": [],
+  "bugs": {
+    "url": "https://github.com/openknowledgebe/equalstreetnames/issues"
+  },
+  "homepage": "https://github.com/openknowledgebe/equalstreetnames#readme",
```
Comment (RobinLinde) [inline website/package.json:17]: Should probably be changed to `https://github.com/EqualStreetNames/equalstreetnames#readme`

File: website/package.json [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  "directories": {
+    "doc": "docs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openknowledgebe/equalstreetnames.git"
+  },
+  "keywords": [],
+  "bugs": {
+    "url": "https://github.com/openknowledgebe/equalstreetnames/issues"
```
Comment (RobinLinde) [inline website/package.json:15]: Should probably be changed to `https://github.com/EqualStreetNames/equalstreetnames/issues`

File: website/package.json [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  "name": "@equalstreetnames/website",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "directories": {
+    "doc": "docs"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/openknowledgebe/equalstreetnames.git"
```
Comment (RobinLinde) [inline website/package.json:11]: Should probably be changed to `git+https://github.com/EqualStreetNames/equalstreetnames.git`

**Comment by jbelien** [comment]
Yes, small changes in all `.html` files, I'll also take care of it once merged :)

And yes, the new "status" of `package.json` makes me really happy 😁

**Review by RobinLinde — APPROVED** [review]
Looks good, as far as I can tell 👍

**Comment by jbelien** [comment]
Update of HTML files and GitHub Actions workflows is done ! 🎉

**Pull request merged**